src/webapi/document.rs

```diff
@@ -67,4 +69,54 @@ impl Document {
             ).into_reference_unchecked()
         }
     }
+
+    /// Returns the <body> or <frameset> node of the current document, or null if no such element exists.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Document/body)
+    // https://html.spec.whatwg.org/#the-document-object:dom-document-body
+    pub fn body( &self ) -> Option< HtmlElement > {
+        unsafe {
+            js!(
+                return @{self}.body;
+            ).into_reference_unchecked()
+        }
+    }
+
+    /// Returns the <head> element of the current document. If there are more than one <head>
+    /// elements, the first one is returned.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Document/head)
+    // https://html.spec.whatwg.org/#the-document-object:dom-document-head
+    pub fn head( &self ) -> Option< HtmlElement > {
+        unsafe {
+            js!(
+                return @{self}.head;
+            ).into_reference_unchecked()
+        }
+    }
+
+    /// Gets the title of the document.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Document/title)
+    // https://html.spec.whatwg.org/multipage/semantics.html#the-title-element
+    pub fn get_title( &self ) -> String {
```

Comment:
Hmm... the issue of whenever we should use `get_foo` or just simply `foo` for the reader is definitely something to consider.

Corrently we're somewhat inconsistent - e.g. in `ImageElement` and in `WebSocket` we **don't** use the `get_` prefix while in `CanvasRenderingContext2d` we do. I guess it'd be nice to pick one and be consistent from now on.

Personally I prefer to **not** use the `get_` prefix, so if you could get rid of it it'd be great. (I'll make this consistent across the whole crate on the next semver bump.)

Reply:
We should also update that in the CONTRIBUTING.md then.

Reply:
wasm-bindgen uses the `foo` and `set_foo` convention, which I personally also like.